HowTos/customize_aws_acls_for_cloudn_dcx.rst

```diff
@@ -0,0 +1,62 @@
+.. meta::
+   :description: Customize AWS ACLs for CloudN DCCX
+   :keywords: AWS, Aviatrix Cloud Interconnect, DCCX, CloudN, datacenter extension
+
+
+=========================================================================
+Customize AWS ACLs Inbound/Outbound Rules for Datacenter Extension Gateway
+=========================================================================
+
+Introduction
+============
+
+This document provides the minimized Network ACLs Requirements for Datacenter Extension Gateway. The customization reduces the scope of AWS ACLs Inbound and Outbound rules 
```

Comment:
This document provides the minimum Network ACLs requirements for Datacenter Extension Gateway. This customization reduces the scope of AWS Network ACLs Inbound and Outbound rules